.travis.yml

```diff
@@ -12,3 +12,12 @@ env:
   global:
   - R_REMOTES_NO_ERRORS_FROM_WARNINGS=true
   - _R_CHECK_SYSTEM_CLOCK_=FALSE
+
+- r: release
+    env:
+      - DEVEL_PACKAGES=true
+    r_github_packages:
```

Comment:
`dance` does not depend on `Rcpp`. Can you instead the remotes for the packages that are the the `Imports` field of the DESCRIPTION file: 

```
Imports: 
    vctrs (>= 0.1.0),
    rlang (>= 0.3.1),
    magrittr (>= 1.5),
    dplyr (>= 0.8.0.1),
    purrr (>= 0.3.1),
    assertthat (>= 0.2.0),
    tidyselect (>= 0.2.5),
    glue (>= 1.3.0),
    tibble (>= 2.0.1),
    zeallot (>= 0.1.0),
    utils
```

Not necessary to include `utils` as this is a base package.

Reply:
apologies. changes now made and the travis job should now check for new development versions of the correct dependencies.